template.tpl

```
@@ -1484,7 +1632,9 @@ scenarios:
     const mockData = {
       eventName: 'Example',
       metricType: 'metric',
-      clientSideId: '1234',
+      rojectKey: 'pkey',
```

Typo: `rojectKey`. Should we also fail if `projectKey` is not set? :smile:

---

Thanks for spotting this! 
There is a non-empty validation check for that field, so it shouldn't allow users to leave it empty, that is why there is no such check in code.